composer.json

```diff
@@ -75,7 +75,7 @@
         "drupal/search_exclude": "2.0.0-beta2",
         "drupal/slick": "2.3",
         "drupal/slick_views": "2.4",
-        "drupal/smart_date": "3.2.0",
+        "drupal/smart_date": "3.3.x-dev",
```

Comment:
Is there a specific fix/patch that's needed for the smart_date module?  We don't typically like using arbitrary dev branch versions for our composer dependencies because it makes our builds less deterministic.